### PR TITLE
Add dotenv support

### PR DESCRIPTION
**Pull request by nivcertora**
https://certora.atlassian.net/browse/CERT-7881

### CODE REVIEW

File: Quorum/config.py [hunk 1]
```diff
@@ -1,9 +1,12 @@
 import os
 import shutil
 from pathlib import Path
+import dotenv
 
 import Quorum.utils.pretty_printer as pp
 
+dotenv.load_dotenv(override=True)
```
Comment (yoav-el-certora) [inline Quorum/config.py:8]: Thinking out loud, should we alert the user on override cases?

Comment (nivcertora) [reply Quorum/config.py:8]: Hmm i guess its a good idea

File: Quorum/.env.example
Comment (yoav-el-certora) [inline Quorum/.env.example:1]: Please add a section regarding this requirement in the README file

File: Quorum/config.py (hunk 1 above)
Comment (nivcertora) [reply Quorum/config.py:8]: Not sure how to do it tbh

Comment (yoav-el-certora) [reply Quorum/config.py:8]: Probably something like - 
for a in .env[vars]:
if a in os.getenviron() : print(something)

Comment (yoav-el-certora) [reply Quorum/config.py:8]: Its not straight forward, but this should work, just a solution from my head without investigating too much

File: Quorum/.env.example
Comment (nivcertora) [reply Quorum/.env.example:1]: Done

File: Quorum/config.py (hunk 1 above)
Comment (vaknin34) [reply Quorum/config.py]: Done

File: README.md [hunk 2]
```diff
@@ -49,6 +49,14 @@ export ANTHROPIC_API_KEY="your_anthropic_api_key"
 
 Replace `your_etherscan_api_key`, `your_anthropic_api_key` with your actual API keys.
 
+Alternatively, you can set these environment variables in a `.env` file in the current working directory where you use the tool:
+
```
Comment (yoav-el-certora) [inline README.md:53]:
```suggestion
(Please review /Quorum/.env.example)

```

**Review by yoav-el-certora — DISMISSED** [review]

**Pull request merged**